lib/chiapos/src/calculate_bucket.hpp

```diff
@@ -60,20 +61,24 @@ std::map<uint8_t, uint8_t> kVectorLens = {
     {8, 0}
 };
 
-// Precomputed shifts that specify which entries match with which other entries
-// in adjacent buckets.
-uint16_t matching_shifts_c[2][kC];
 
-// Performs the precomputation of shifts.
-void precompute_shifts() {
+auto load_tables()
+{
+    std::array<std::array<std::array<uint16_t, kExtraBitsPow>, kBC>, 2> L_targets{};
```

Comment:
What's the reason for using std::array instead of a normal array?

Reply:
that function used to be constexpr, but CI was failing to compile due to limited memory. 
Since it's not constexpr, it can be changed to normal array now.